### ISSUE TITLE
Optimizer for unet model for Pascal voc segmentation

### ISSUE THREAD

**Issue by saiphanish7** [meetshah1995/pytorch-semseg#238]
Can you specify me the optimizer required for unet model on Pascal Voc segmentation using focal loss. Should I have to use any learning rate schedulers?
